app/models/customer.rb

```diff
@@ -80,6 +80,12 @@ def editable?
       wallets.none?
   end
 
+  def preferred_document_locale
+    return document_locale&.to_sym if document_locale.present?
+
+    organization.document_locale&.to_sym
```

Comment:
```suggestion
    organization.document_locale.to_sym
```

as organization.document_locale cannot be null?

Reply:
fixed